### ISSUE TITLE
Failed to install dotnet-sdk-3.1 on ubuntu 21.04

### ISSUE THREAD

**Issue by netroby** [dotnet/sdk#16900]
I am testing dotnet install on ubuntu 21.04

```
$ sudo apt install dotnet-sdk-3.1           
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 dotnet-sdk-3.1 : Depends: dotnet-targeting-pack-3.1 (>= 3.1.0) but it is not installable
                  Depends: netstandard-targeting-pack-2.1 (>= 2.1.0) but it is not installable
                  Depends: aspnetcore-targeting-pack-3.1 (>= 3.1.10) but it is not installable
E: Unable to correct problems, you have held broken packages.

```
but failed to install dotnet-sdk-3.1 , due to some package missing.

I read the document and operate as the guide told me :

https://docs.microsoft.com/en-us/dotnet/core/install/linux-ubuntu#2010-

**Comment by marcpopMSFT** [comment]
@leecow @rbhanda is 21.04 published?

**Comment by rbhanda** [comment]
Hi @marcpopMSFT We have not published packages to 21.04. Our May update will publish the packages to the distro.
